5.3 release blog post

jenweber opened the pull request:
This PR has the boilerplate for the 5.3 release blog post. Volunteer help is needed to fill in the details. Thanks in advance! If you have push access to this repo, feel free to push to this branch as you work, or open a PR against this branch. Reviewers are also welcome to help merge code suggestions.

## Target release window
Week of Sept 18

## Volunteer writers needed

Please comment on this PR if you can help out! You can look at the latest beta and changelog for each library to see what to cover.

- [x] Write ember.js section
- [x] Write ember-cli section
- [x] Write ember-data section

## Team approvals

- [ ] ember.js
- [ ] ember-cli
- [ ] ember-data
- [ ] typescript team

jenweber committed:
Set up 5.3 release blog post

kategengler commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,118 @@
+
+Today the Ember project is releasing version 5.3 of Ember.js, Ember Data, and Ember CLI. <!-- Block start: Uncomment if an LTS candidate --><!--This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.--><!-- Block end -->
+
+This release kicks off the 5.3.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
```suggestion
This release kicks off the 5.4 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

kategengler commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,118 @@
+
+#### Deprecations
+
+Ember.js 5.3 introduced 0 deprecations.
```
```suggestion
Ember.js 5.3 introduced 1 deprecation.
```

kategengler commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,118 @@
+#### Deprecations
+
+Ember.js 5.3 introduced 0 deprecations.
+
```
```suggestion
1. Deprecate implicit record loading in Ember Route per [RFC #774](https://rfcs.emberjs.com/id/0774-implicit-record-route-loading). Please see [the deprecation guide](https://deprecations.emberjs.com/v5.x#toc_deprecate-implicit-route-model) for more information.
```

kategengler commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,118 @@
+
+#### Features
+
+Ember.js 5.3 introduced 2 features.
```
```suggestion
Ember.js 5.3 introduced 0 features.
```

Possibly this block and the above should be removed?

jenweber replied:
Typically I haven't deleted the 0 count sections, but that seems great to do. Removed.

jenweber committed:
Apply Katie's section and feedback

Co-authored-by: Katie Gengler <katie@kmg.io>

jenweber committed:
Lint fix

jenweber committed:
Update casing for Ember Data -> EmberData

This aligns with the latest guidance from package authors and
contributors. I updated our blog template.

Reference the README of https://github.com/emberjs/data
when in doubt.

jenweber committed:
Copy paste changelog notes for EmberData

jenweber committed:
Add Ember CLI content based on release notes

jenweber commented:
@gitKrystan let me know if anything should be said in this release about TypeScript. Thank you!

runspired commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,128 @@
+
+#### Features
+
+EmberData 5.3 introduced 3 features.
```
```suggestion
EmberData 5.3 introduced several new features.
```

runspired replied:
The release notes was a pretty high level overview, if we want to get into brass tacks:

- the 3 cache methods
- 5 builders each for REST/ActiveRecord/JSON:API
- 2 serialization utils for JSON:API
- 6 builder utils
- a basic lifetimes service

Depending on how you're counting thats as many as 17 new features :D

gilest replied:
And a partridge in a pear tree

runspired reviewed (approved):
overall looks good, left one comment

bertdeblock commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,128 @@
+
+Ember CLI 5.3 introduced 1 deprecation.
+
+- Deprecate outputPaths build option, as output paths will no longer be predetermined under Embroider
```
Can we review / merge the guide as well => https://github.com/ember-learn/deprecation-app/pull/1337 ?

jaredgalanis replied:
Done ✅

jaredgalanis committed:
Update content/ember-released-5-3.md

Co-authored-by: Chris Thoburn <runspired@users.noreply.github.com>

NullVoxPopuli commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,128 @@
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
```
in this ember-cli release, pnpm support is no longer behind a feature flag, so folks can use `--pnpm`

NullVoxPopuli replied:
(what's the proper boilerplate for new features?)

jaredgalanis replied:
@NullVoxPopuli I can add it

jaredgalanis replied:
> (what's the proper boilerplate for new features?)

https://github.com/ember-learn/ember-blog/blob/49eea7a790ff6deba6805e5bbf88db8c4b3cb380/post-templates/emberblog-template.md

NullVoxPopuli reviewed (changes requested):
Blocking on need of the addition of mentioning official pnpm support
(and maybe mentioning pnpm has worked with ember already for quite some time, but ember-cli didn't know about pnpm)

kategengler commented on content/ember-released-5-3.md:
```diff
@@ -0,0 +1,128 @@
+
+#### Deprecations
+
+EmberData 5.3 introduced 5 deprecations.
```
Can we get this merged? https://github.com/ember-learn/deprecation-app/pull/1339

jaredgalanis replied:
@kategengler Yes, I'll get it done today.

jaredgalanis replied:
✅

jaredgalanis committed:
add pnpm feature to cli

jaredgalanis commented:
merging as all feedback was met
